docs/modifiers.md

```diff
@@ -0,0 +1,225 @@
+# InterPro API documentation: modifiers
+
+This document aims to provide some guidance on how to use the InterPro API modifiers for the different API end points blocks, together with examples. Modifiers allow to filter and/or order the data returned by the API call.
+
+## Apply to any API call
+
+| Modifier                         | Data returned                                     | Example                                                               |
+| -------------------------------- | ------------------------------------------------- | --------------------------------------------------------------------- |
+| page_size=< _number up to 200_ > | Number of results returned at a time (default=20) | https://www.ebi.ac.uk:443/interpro/api/protein/reviewed?page_size=100 |
+| search=< _text_ >                | Entries matching the text search                  | https://www.ebi.ac.uk:443/interpro/api/taxonomy/uniprot/?search=9606  |
+
+## /api/entry
+
+| Modifier                                                 | Data returned                                                                                                                  | Example                                                                                        |
+| -------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------- |
+| group_by=type                                            | Number of entries for each entry type (e.g. family, domain, site...)                                                           | https://www.ebi.ac.uk:443/interpro/api/entry?group_by=type                                     |
```

Comment:
```suggestion
| `group_by=type`                                            | Number of entries for each entry type (e.g. family, domain, site...)                                                           | https://www.ebi.ac.uk:443/interpro/api/entry?group_by=type                                     |
```
I think I would use the markdown notation for `code` on all the modifiers.
It might be just a matter of a personal preference, so feel free to ignore.